src/worker.js

```diff
@@ -7,20 +7,17 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
-export default (app, config, onServer, workerId) => {
-  let server;
-
-  // ===== Middleware =========================================================
+export const attachMiddleware = (app, config) => {
```

Comment:
Hmm, maybe I push to the wrong branch :) I did on the train when it was arriving to my station. :) Let me check.